### PR TITLE
Initial name swapping

### PR DESCRIPTION
**Pull request by garettarrowood**
WIP

### CODE REVIEW

**Comment by garettarrowood** [comment]
We decided to include the O in the repo name. Simply because cobra is a lot easier to remember than cbra. I'll change the executable.  And the main module to Cobra.

**Comment by benlangfeld** [comment]
You should change the main module to CobraCommander to match the gem name, including renaming lib/cbra. This lines everything up for Bundler's assumptions of autoloading.

**Comment by garettarrowood** [comment]
Executable should still be `cobra` though, and not `cobra-commander`, correct?

**Comment by benlangfeld** [comment]
> Executable should still be `cobra` though, and not `cobra-commander`, correct?

Sure.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+# cobra_commander
 
-[![Gem](https://img.shields.io/gem/dv/cbra/stable.svg)](https://rubygems.org/gems/cbra)
-[![Gem](https://img.shields.io/gem/v/cbra.svg)](https://rubygems.org/gems/cbra)
-[![Travis](https://img.shields.io/travis/powerhome/cbra.svg)](https://travis-ci.org/powerhome/cbra)
-[![Code Climate](https://img.shields.io/codeclimate/github/powerhome/cbra.svg)](https://codeclimate.com/github/powerhome/cbra)
-[![Gemnasium](https://img.shields.io/gemnasium/powerhome/cbra.svg)](https://gemnasium.com/github.com/powerhome/cbra)
+[![Gem](https://img.shields.io/gem/dv/cobra_commander/stable.svg)](https://rubygems.org/gems/cobra_commander)
+[![Gem](https://img.shields.io/gem/v/cobra_commander.svg)](https://rubygems.org/gems/cobra_commander)
+[![Travis](https://img.shields.io/travis/powerhome/cobra.svg)](https://travis-ci.org/powerhome/cobra_commander)
```
Comment (benlangfeld) [inline README.md]: Still `cobra` here?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+[![Code Climate](https://img.shields.io/codeclimate/github/powerhome/cobra_commander.svg)](https://codeclimate.com/github/powerhome/cobra_commander)
+[![Gemnasium](https://img.shields.io/gemnasium/powerhome/cobra_commander.svg)](https://gemnasium.com/github.com/powerhome/cobra_commander)
 
-Tools for working with Component Based Rails Apps (see http://shageman.github.io/cbra.info/). Includes tools for graphing the components of an app and their relationships, as well as selectively testing components based on changes made.
+Tools for working with Component Based Rails Apps (see http://shageman.github.io/cobra.info/). Includes tools for graphing the components of an app and their relationships, as well as selectively testing components based on changes made.
```
Comment (benlangfeld) [inline README.md]: Oops. Still should be `http://shageman.github.io/cbra.info`.

**Pull request merged**